pipelines/rh-advisories/rh-advisories.yaml

```diff
@@ -444,7 +449,8 @@ spec:
         - name: data
           workspace: release-workspace
       runAfter:
-        - push-rpm-data-to-pyxis
+        - collect-pyxis-params
+        - collect-data
```

Comment:
collect-pyxis-params has runAfter collect-data so I don't think this is needed

Reply:
OK, removed.